src/cljc/jinteki/decks.cljc

```diff
@@ -220,8 +213,34 @@
                                           cycle
                                           (first (sort (mapv #(parse-date (:available %)) sets-in-cycle)))))
                                       {} cycles)
-        valid-cycles (map first (take-last 2 (sort-by last (filter (fn [[cycle date]] (before-today? date)) cycle-release-date))))]
-    (map :name (filter (fn [set] (some #(= (:cycle set) %) valid-cycles)) sets))))
+        valid-cycles (map first (take-last n (sort-by last (filter (fn [[cycle date]] (before-today? date)) cycle-release-date))))]
+    valid-cycles))
+
+(defn sets-in-newest-cycle
+  "Returns sets in the newest cycle of released datapacks - for Modded format"
+  [sets]
+  (map :name (filter (fn [set] (some #(= (:cycle set) %) (get-newest-cycles sets 1))) sets)))
+
+(defn sets-in-two-newest-cycles
```

Comment:
I'd combine these two functions to `sets-in-newest-n-cycles" and pass in `n` as well.